base/applications/network/ping/ping.c

```diff
@@ -425,14 +428,20 @@ Ping(void)
 
     if (RequestSize != 0)
     {
-        SendBuffer = malloc(RequestSize);
+        SendBuffer = (PUCHAR)malloc(RequestSize);
```

Comment:
```suggestion
        SendBuffer = malloc(RequestSize);
```
In C the cast isn't needed.